source/Halibut.Tests/TestServices/FileTransferService.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.IO;
+using Octopus.Tentacle.Contracts;
+
+namespace Halibut.Tests.TestServices
+{
+    public class FileTransferService : IFileTransferService
```

Comment:
This has been moved into the Halibut test repo so it can depend on the latest version of DataStream so that it can make use of the async writer.